website/docs/cli/commands/state/show.mdx

```diff
@@ -69,7 +69,7 @@ $ terraform state show 'packet_device.worker[0]'
 ## Example: Show a Resource configured with for_each
 
 The example below shows the `"example"` instance of a `packet_device` resource named `worker` configured with
```

Comment:
Thank you! I have a couple of style notes. First, we're trying to move away from positional language like "below", so could you change that to "The following example"? And then I've just reworked your addition a little bit to help it flow better. Do you feel comfortable making this change?

"The following example shows the `"example"` instance of a `packet_device` resource named `worker` configured with [`for_each`](/language/meta-arguments/for_each). You must place the resource name in single quotes when it contains special characters (e.g., double quotes)."

Reply:
Thanks Laura, done 👍🏽